python/tests/tornado/test_hyperbahn.py

```diff
@@ -57,8 +58,21 @@ def test_request():
 
 
 @pytest.mark.gen_test
-def test_register():
+def test_advertise(tchannel_server):
+    endpoint = b'ad'
+
+    f = Future()
+
+    @tornado.gen.coroutine
+    def execution(request, response):
+        body = yield request.get_body()
+        f.set_result(body)
```

Comment:
You'll want to send some sort of dummy response here. Just an empty dictionary will do.